testing/libgeotiff/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=libgeotiff
+pkgver=1.4.2
+pkgrel=0
+pkgdesc="TIFF based interchange format for georeferenced raster imagery."
+url="https://trac.osgeo.org/geotiff"
+arch="all"
+license="Public Domain"
+options="!check"
+depends="libgcc libstdc++ musl"
+makedepends="autoconf automake libtool tiff-dev proj4-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::http://download.osgeo.org/geotiff/libgeotiff/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+        mkdir -p "$builddir"
```

Comment:
why? if you actually create the directory, then will the build bellow fail because new directories are empty.